README.md

```diff
@@ -34,7 +34,7 @@ By default, users are managed both by CouchDB and LDAP.
 
 1. In CouchDB, [create an administrator account](http://localhost:5984/_utils/#createAdmin).
 2. In CouchDB, [create a database](http://localhost:5984/_utils/#/_all_dbs) named `_users`.
-3. Choose a password and set in as the `secret` in `conf/couchdb.ini` and `forwardedSecretLogin`.
+3. Choose a password and set in as the `secret` in `conf/couchdb.ini` and `conf/aaaforrest.yml`.
```

Comment:
It's a good idea to mention the file. 
And it is true that it is called `secret` in both files.
I hope people will understand that `secret` is not `session.secret`.